docs/experiment/experiment_creation.md

```diff
@@ -0,0 +1,75 @@
+# Experiment Creation
+
+Start by [navigating](../project/project_introduction.md#project-navigation) 
+to the project summary view of your project of interest.
+To create an experiment in the selected project, click the add button within the experiment component on the top right.
+![project_summary](../project/images/project_summary.png)
+
+!!! info "Project role"
+    Should you not see the add button,
+    ensure that you have been granted the "write" or "admin" project role by the project owner/admin!
+
+This will open the create experiment dialog, in which you can specify the name of the experiment and the included species, specimen and analyte
+information of the involved organisms.
+To provide these details you can select one or more entries via the provided combo boxes.
```

Comment:
Is "combo box" a term everyone understands? These are not "really" combo boxes anymore, either. If you decide to change it, please also do so in the project creation documentation.

Reply:
Good point, i changed it to selection field 🧠